### PR TITLE
Refactor structure for next needs

### PR DESCRIPTION
**Pull request by chalasr**

### CODE REVIEW

File: src/Env/Config/Manala.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * (c) Manala <contact@manala.io>
+ *
+ * For the full copyright and license information, please refer to the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Manala\Env\Config;
+
+/**
+ * Make.
```
Comment (ogizanagi) [inline src/Env/Config/Manala.php]: Make ? Make what ? 😆 

EDIT: oh, just saw it was simply a copy/paste from above 😅 . Anyway may be removed :)

Comment (chalasr) [reply src/Env/Config/Manala.php]: @ogizanagi: The biggest troll ever 😆 

It corresponds to the unix`make` utility, same as `Vagrant` for `vagrant` and `Ansible` for `ansible`. But you're right it's misleading, I'll expand this description

Comment (ogizanagi) [reply src/Env/Config/Manala.php]: @chalasr : No no no. I mean... unless `make` has something to do with the `Manala` class on which this docblock is... :trollface:

Comment (ogizanagi) [reply src/Env/Config/Manala.php]: But appart from this, I'm not fan of such docblocks simply repeating the class name without bringing more information, indeed. :)

Comment (chalasr) [reply src/Env/Config/Manala.php]: lol, sorry about the confusion :trollface:

**Pull request merged**